lib/calendar.dart

```diff
@@ -1,36 +1,42 @@
 import 'package:flutter/material.dart';
 import 'package:table_calendar/table_calendar.dart';
+import 'First_Screen.dart';
 
-void main() {
-  runApp(MyApp());
-}
+// void main() {
+//   runApp(MyApp());
+// }
 
-class MyApp extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        primarySwatch: Colors.blue,
-      ),
-      home: MyHomePage(title: 'Inkling'),
-    );
-  }
-}
```

Comment:
Yoshi and I commented out this section because I think its redundant and possibly conflicting with the main.dart The page renders fine without it.